include/SendResult.h

```diff
@@ -34,21 +34,23 @@ enum SendStatus {
 class ROCKETMQCLIENT_API SendResult {
  public:
   SendResult();
-  SendResult(const SendStatus& sendStatus, const std::string& msgId,
+  SendResult(const SendStatus& sendStatus, const std::string& msgId, const std::string& offsetMsgId, 

```

Comment:
It is better to overload its constructor. keep the old method and continue to use.

Reply:
SendResult constructed by SDK and used by producer, so it doesn't matter to remove old method because there will not use old method